SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +57,18 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		// There is no need to encrypt UnrealWorker (server) to runtime traffic as these machines will always be in the same data center.
+		if (*Config.WorkerType != SpatialConstants::DefaultServerWorkerType.ToString() && GetDefault<USpatialGDKSettings>()->bUseSecureConnection)
```

Comment:
Sorry to be a pain, but can we also expose the option to explicitly set all worker types to be secure? If you are sending sensitive data you will still want secure networking in the datacentre, as the network is shared by all sorts of GCP customers.

I think the default should be insecure all, then the next most common would be only secure clients, but we should also have an option to have everything secure.

Reply:
Done.